larray/core.py

```diff
@@ -1101,6 +1100,9 @@ def group(self, *args, **kwargs):
             return key.__class__(key.key, name, self)
         return LGroup(key, name, self)
 
+    def group(self,  *args, **kwargs):
+        raise NotImplementedError("Axis.group is deprecated. Use the syntax \"age[10:19] >> 'teenagers'\" instead.")
```

Comment:
it would be nicer for the user to use the actual key/axis he used in the error message instead of age[10:19]